src/libsync/account.cpp

```diff
@@ -147,7 +147,21 @@ void Account::setAvatar(const QImage &img)
 
 QString Account::displayName() const
 {
-    QString dn = QString("%1@%2").arg(credentials()->user(), _url.host());
+    const auto userName = [this]() {
+        if (!_davUser.isEmpty()) {
+            return _davUser;
```

Comment:
this is wrong for something that will be a display name
`_davUser` is the user backend internal id coming from server and should never be display to teh user